lib/irb/source_finder.rb

```diff
@@ -26,6 +31,13 @@ def find_source(signature)
       when /\A(?<owner>[A-Z]\w*(::[A-Z]\w*)*)#(?<method>[^ :.]+)\z/ # Class#method
         owner = eval(Regexp.last_match[:owner], context_binding)
         method = Regexp.last_match[:method]
+        if esses
```

Comment:
That's use `s_count` for this condition?

Reply:
We can use it, it will probably be clearer that way.